crates/ansi-to-html/src/html/minifier.rs

```diff
@@ -51,6 +51,13 @@ pub(crate) struct Minifier {
 }
 
 impl Minifier {
+    pub fn new(var_prefix: Option<String>) -> Self {
```

Comment:
Please make methods that aren't part of the public API `pub(crate)` to make it clear.